tests/unit/helpers/test_cli.py

```diff
@@ -79,7 +80,30 @@ def test_create_artifact_payload_from_filepath(
         }
 
         assert (
-            create_artifact_payload_from_filepath(
+            create_artifact_add_payload_from_filepath(
+                filepath=temp_file,
+                path=path,
+            )
+            == expected_artifact_payload
+        )
+
+    def test_create_artifact_delete_payload_from_filepath(
+        self,
+        temp_file: str,
+    ) -> None:
+        """
+        Test that the artifact payload is created correctly given the
+        filepath of the artifact
+        """
+
+        path = "/fake/path/"
+
+        expected_artifact_payload = {
+            "targets": [f"{path}{temp_file.split('/')[-1]}"]
+        }
+
+        assert (
+            create_artifact_delete_payload_from_filepath(
```

Comment:
Would be better and more readable if you assign a variable `result` to the `create_artifact_delete_payload_from_filepath` call and check it with an assert.